market_nextdoor_api/tests/views_tests/test_markets_location.py

```diff
@@ -0,0 +1,124 @@
+from rest_framework.test import APITestCase
+from django.urls import reverse
+from rest_framework import status
+from unittest.mock import patch
+import responses
+import os
+import pdb
+
+class MarketLocationsTestCase(APITestCase):
+  def setUp(self):
+    pdb
```

Comment:
There's a left over pdb in the test. Will this stop the test from running?

Reply:
removing it right now! It ran with it, but I think that's because my snippet isn't working. Either way I removed it because it's unecessary.